.github/workflows/issue_prioritization.yml

```diff
@@ -62,8 +71,8 @@ jobs:
               }
             }' -f project=$PROJECT_ID -f issue=$ISSUE_ID > issue_data.json
           
-          echo 'ITEM_ID='$(jq '.data.addProjectNextItem.projectNextItem.id' issue_data.json) >> $GITHUB_ENV
-          echo 'ITEM_CREATION_DATE='$(jq '.data.addProjectNextItem.projectNextItem.content.createdAt' issue_data.json) >> $GITHUB_ENV
+          echo 'ITEM_ID='$(jq '.data.addProjectV2ItemById.item.id' issue_data.json) >> $GITHUB_ENV
+          echo 'ITEM_CREATION_DATE='$(jq '.data.addProjectV2ItemById.item.content.createdAt' issue_data.json) >> $GITHUB_ENV
```

Comment:
Is there a reason why we are removing characters 2-11 from the creation date in the prebid server issue prioritization workflow but not in the prebid cache workflow?
In the prebid server file, I see that the `createdAt` field is piped to a `cut` command:
`echo 'ITEM_CREATION_DATE='$(jq '.data.addProjectV2ItemById.item.content.createdAt' issue_data.json | cut -c 2-11) >> $GITHUB_ENV`

Reply:
Good catch @bsardo . Fixed.